process_manager/process_manager_interface.py

```diff
@@ -6,32 +6,33 @@
 
 from django.conf import settings
 from drunc.process_manager.process_manager_driver import ProcessManagerDriver
-from drunc.utils.shell_utils import DecodedResponse, create_dummy_token_from_uname
+from drunc.utils.shell_utils import DecodedResponse
 from druncschema.process_manager_pb2 import (
     LogRequest,
     ProcessInstanceList,
     ProcessQuery,
     ProcessUUID,
 )
+from druncschema.token_pb2 import Token
 
 
-def get_process_manager_driver() -> ProcessManagerDriver:
+def get_process_manager_driver(username: str) -> ProcessManagerDriver:
     """Get a ProcessManagerDriver instance."""
-    token = create_dummy_token_from_uname()
+    token = Token(token=f"{username}-token", user_name=username)
```

Comment:
Assuming this actually works with their authentication system

Reply:
For the moment they're using a `DummyAuthoriser` that doesn't check the token.